tests/devices/i13_1/test_merlin.py

```diff
@@ -97,7 +96,7 @@ async def test_can_decribe_collect(merlin: Merlin, one_shot_trigger_info: Trigge
     assert (await merlin.describe_collect()) == {
         "merlin": {
             "source": "mock+ca://BL13J-EA-DET-04HDF5:FullFileName_RBV",
-            "shape": [20, 10],
+            "shape": [1, 20, 10],
```

Comment:
FYI @douglaswinter, this may affect downstream reading of documents for i13-1 I suspect.